src/test/regress/expected/isolation_metadata_sync_vs_all.out

```diff
@@ -0,0 +1,595 @@
+unused step name: s3-debug
+Parsed test spec with 3 sessions
+
+starting permutation: s1-begin s2-begin s1-start-metadata-sync s2-alter-table s1-rollback s2-rollback s3-compare-snapshot
+create_distributed_function
```

Comment:
I think not a critical thing, but seeing `create_distributed_function` on each permutation felt unexpected at first. I guess, it shows up because it is the last step of the setup.  Can we somehow  avoid this?

Reply:
I went over the statements in the setup phase, and the outputs are more clear now.